examples-gcc/peripheral_console_liteos/src/gen_os_impl.c

```diff
@@ -317,6 +308,8 @@ const gen_os_driver_t *os_impl_get_driver(void)
 
 void OsSysTickTimerInit(UINT32 reloadValue)
 {
+    portNVIC_CCR_REG = 0x200;//remove div 0 and unalign falut error;
+
```

Comment:
Is the value correct? Why mask these errors?

![image](https://github.com/user-attachments/assets/71a1415e-c869-4c58-a2ab-8e69fe71e346)